src/Microsoft.VisualStudio.Threading/InternalUtilities.cs

```diff
@@ -85,7 +85,7 @@ internal static IEnumerable<string> GetAsyncReturnStackFrames(this Delegate cont
                 stateMachine.GetType().FullName,
                 state,
                 AsyncReturnStackPrefix,
-                (int)GetAddress(stateMachine)); // the int cast allows hex formatting
+                (long)GetAddress(stateMachine)); // the int cast allows hex formatting
```

Comment:
Update the comment too?
```suggestion
                (long)GetAddress(stateMachine)); // the long cast allows hex formatting
```